src/Core/Application/Service/SettingManager.php

```diff
@@ -51,7 +52,7 @@ public function setSettings(array $pathValues): array
             if (isset($pathValues[$settingDefinition->getPath()])) {
                 $setting = $this->buildPathValue($settingDefinition, $pathValues[$settingDefinition->getPath()]);
 
-                if ($settingDefinition->isProject()) {
+                if ($settingDefinition->getSettingType() !== Setting::SETTING_TYPE_SDK) {
```

Comment:
we can leave this method and add `isSdk` is Setting and put this condition there

Reply:
done